src/twenty-three-video/twenty-three-video.js

```diff
@@ -0,0 +1,113 @@
+//# sourceURL=twenty-three-video.js
+import React, { useEffect, useState } from "react";
+// import "./twenty-three-video.scss";
+// import PropTypes from "prop-types";
+import BaseSlideExecution from "../base-slide-execution";
+import { ThemeStyles } from "../slide-util";
+import GlobalStyles from "../GlobalStyles";
+
+/**
+ * TwentyThreeVideo component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.slide The slide.
+ * @param {object} props.content The slide content.
+ * @param {boolean} props.run Whether or not the slide should start running.
+ * @param {Function} props.slideDone Function to invoke when the slide is done playing.
+ * @returns {object} The component.
+ */
+function TwentyThreeVideo({ slide, content = {}, run, slideDone }) {
+  // Content from content
+  const { videoList = "69389106, 74774561" } = content;
+
+  const formattedVideoList = videoList.replaceAll(".", ",").split(",").map((item) => item.trim()).filter(Boolean);
+  const [videoId, setVideoId] = useState(formattedVideoList[0]);
+
+  const lastVideoInPlaylist =
+    formattedVideoList.indexOf(videoId) === formattedVideoList.length - 1;
+
+  useEffect(() => {
+
+    window.addEventListener(
+      "message",
+      function (e) {
```

Comment:
Add this function to a const first, then pass it to addEventListener so that you have a reference you can pass when you remove it below.